test/benchmark/tests/utils/__init__.py

```diff
@@ -0,0 +1,274 @@
+import json
+import logging
+import os
+import re
+import subprocess
+import sys
+import time
+
+import boto3
+import docker
+import git
+import pytest
+import yaml
+from botocore.exceptions import ClientError
+from invoke import run
+from invoke.context import Context
+
+DEFAULT_REGION = "us-west-2"
+IAM_INSTANCE_PROFILE = "EC2Admin"
+S3_BUCKET_BENCHMARK_ARTIFACTS = "s3://torchserve-model-serving/benchmark_artifacts"
+
+DEFAULT_DOCKER_DEV_ECR_REPO = "torchserve-benchmark"
+DEFAULT_DOCKER_DEV_ECR_TAG = "dev-image"
+DEFAULT_DOCKER_DEV_ECR_REPO_TAG = f"{DEFAULT_DOCKER_DEV_ECR_REPO}:{DEFAULT_DOCKER_DEV_ECR_TAG}"
+ECR_REPOSITORY_URL = "{}.dkr.ecr.{}.amazonaws.com/{}"
+
+GPU_INSTANCES = ["p2", "p3", "p4", "g2", "g3", "g4"]
+
+# DLAMI with nVidia Driver ver. 450.119.03 (support upto CUDA 11.2), Ubuntu 18.04
+AMI_ID = "ami-0ff137c06803a8bb7"
+# AMI_ID = "ami-0198925303105158c", with apache2-utils installed
+
```

Comment:
Can this be made a part of config ?

Reply:
The objective is that __init__.py functions as a config. However, yes, it makes sense given the other configs used. Noted here: https://github.com/pytorch/serve/issues/1067